_ml-commons-plugin/connectors.md

```diff
@@ -91,45 +90,31 @@ When enabled, the `backend_roles`, `add_all_backend_roles`, or `access_model` op
 }
 ```
 
-## Creating a connector
-
-You can build connectors in two ways:
-
-1. A **standalone connector**, saved in a connector index, can be reused and shared with multiple remote models but requires access to both the model and the third party being accessed by the connector, such as OpenAI.
+### Node settings
 
-2. An **internal connector**, saved in the model index, can only be used with one remote model. Unlike a standalone connector, users only need access to the model itself to access an internal connector because the connection is established inside the model.
+Standalone connectors consume less resources. Therefore, you can deploy any model from a standalone connector using data nodes. To make sure that your standalone connection uses data nodes, set `plugins.ml_commons.only_run_on_ml_node` to `false`, as shown in the following example:
```

Comment:
`Standalone connectors consume less resources` this looks confusing, should be `Remote model consumes less resources`. Connector can't run independently.  This also give user wrong hints that `internal connector` in remote model consumes more resources.

Reply:
If we want to highlight connector , we can say `Remote model based on connector consumes less resources`